.github/workflows/build_and_test.yml

```diff
@@ -200,18 +200,23 @@ jobs:
       run: |
         cd pytket
         pip install -e . -v
+    - name: Run doctests
+      run: |
+        cd pytket
+        pip install pytest
+        pytest --doctest-modules pytket
     - name: Test pytket with coverage
       if: matrix.os == 'ubuntu-22.04' && matrix.pyver == '3.8'
       run: |
         cd pytket/tests
         pip install -r requirements.txt
-        pytest --ignore=simulator/ --doctest-modules --hypothesis-seed=1 --cov=../pytket --cov-branch --cov-report=html --cov-report=xml:htmlcov/cov.xml
```

Comment:
Were there any of these doctests in the pytket/tests directory before? Or was this --doctest-modules redundant before this PR?

Reply:
No, it was doing nothing. I'm not sure if it ever worked; I suspect not; in any case there were no doctests for it to test.